### ISSUE TITLE
UI: when following wrong link from edge to vertex in nonexisting collection misleading error is printed

### ISSUE THREAD

**Issue by gamedev8** [arangodb/arangodb#2668]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [x] 3.1

On this operating system:
  - [x] Ubuntu .deb

### this is an AQL-related issue:
- [x] I'm using graph features

I'm issuing AQL via:
- [x] web interface with this browser:     running on this OS:

These are the steps to reproduce:
1) create a document and make sure you put a colon in the _key (eg. document::1)
2) create another document, but don't put a colon in the _key this time (eg. document2)
3) create an edge collection
4) add an edge to the collection and connect it to the documents you created in step 1 and 2

The following problem occurs: The edge will not be connected to the document with the colon in the _key field. You won't be able to retrieve it in your queries.

**Comment by dothebart** [comment]
I've tried:

    FOR x IN OUTBOUND 'a/document::1' knows RETURN x

And got:
```
[
  {
    "_key": "document2",
    "_id": "a/document2",
    "_rev": "_VMMQEC2--A"
  }
]
```

I've tried both latest 3.1and devel community edition - both worked. 

Can you provide a step by step js instruction list reproducing this? Aka
```
db._create(... 
db._save(...
db._query( ...
```

**Comment by gamedev8** [comment]
Hmm I'm not able to repro the data issue right now, but I'm still able to repro this one. Navigate to the edge document in the Web UI, then click on the ID that has the colon in it, and it will go to a blank page.

So when you click on the ID ...
![arango1](https://user-images.githubusercontent.com/1688049/27670488-3bf7298a-5c5b-11e7-835d-b2b42c1aef18.png)


It goes here ...
![arango2](https://user-images.githubusercontent.com/1688049/27670543-9e6a5808-5c5b-11e7-9f69-3083e3f90648.png)

**Comment by dothebart** [comment]
The `_from` and `_to` are links to documents in the `person` collection. However, the `knows_graph` names its vertex collection `persons` - thus the link tries to fetch a document in a non existing collection.

The UI issues the error message: `Error: Could not detect collection type` - Instead it should revalidate whether the collection exists and create an appropriate error.

The link could also use a mouse over ala `jump to vertex document`.

**Comment by hkernbach** [comment]
Thanks for the report. Fixed in 3.2 and devel branches. 
PRs: https://github.com/arangodb/arangodb/pull/3874 & https://github.com/arangodb/arangodb/pull/3873

Will then be available in 3.2.10 and 3.3

**Comment by sleto-it** [comment]
Hi,

Thanks again for opening this ticket. Version 3.2.10 has been released yesterday

I am now closing this ticket. Please feel free to comment if the problem persists

Many thanks,
